pydicom/encaps.py

```diff
@@ -22,7 +22,7 @@
 # If a single frame, it may be 0 length/no value,
 # or it may have a single pointer (0).
 
-from pydicom.config import logger
+import pydicom.config
```

Comment:
Keep the previous call. Otherwise the calls become unreadable. 
```py
logger.debug ( "bla bla bla") 
```
is much more readable than 
```py
pydicom.config.logger.debug
```
If you have more than one logger, it might make sense. But I would still prefer something like:

```py
from moduleA.subA.sub.sub import logger as Alogger
from moduleB.sub.sub import logger as Blogger
```